server.go

```diff
@@ -230,6 +230,17 @@ type Config struct {
 	//
 	// If unset or nil, the group aggregation feature will be disabled on the server.
 	GroupAggregator GroupAggregator
+
+	// JanitorInterval specifies the average interval of janitor checks for expired completed tasks.
+	//
+	// If unset or zero, default interval of 8 seconds is used.
+	JanitorInterval time.Duration
+
+	// JanitorBatchSize specifies the number of expired completed tasks to be deleted in one run.
+	//
+	// If unset or zero, default batch size of 100 is used.
+	// Make sure to not put a big number as the batch size to prevent a long-running script.
+	JanitorBatchSize int
```

Comment:
> // Make sure to not put a big number as the batch size to prevent a long-running script.

Perhaps we should also emit a WARN log when an excessively large batch size is passed in the `Server` constructor (Maybe anything > 100 as per the original spec).